lib/internal/bootstrap_node.js

```diff
@@ -443,7 +443,7 @@
   if (EXPOSE_INTERNALS) {
     NativeModule.nonInternalExists = NativeModule.exists;
 
-    NativeModule.isInternal = function(id) {
```

Comment:
This is best left unchanged since the other `isInternal()` implementation *does* use `id` and we won't know which is being used at runtime. IIRC V8 still prefers it (performance-wise) when the argument/parameter counts match.